securedrop/source_templates/index.html

```diff
@@ -54,12 +76,14 @@
     Included here so the images can preload while the user is first
     reading the page. Hidden by default. -->
     <div class="bubble">
-      <p>You appear to be using the Tor Browser. You can disable Javascript in 3 easy steps!</p> 
+      <p>You appear to be using the Tor Browser. You can turn the Security Slider to High in 4 easy steps!</p>
       <ol>
-        <li>Click the <img src="static/i/no16.png" alt="NoScript icon" /> NoScript icon in the toolbar above</li>
-        <li>Click <strong><img src="static/i/no16-global.png" alt="Forbid Scripts Globally" /> Forbid Scripts Globally (advised)</strong></li>
-        <li>If the page does not refresh automatically, <a href="/">click here</a> to refresh the page</li> 
+        <li>Click the <img src="{{ url_for('static', filename='i/toronion.png') }}" alt="Tor icon" />Tor icon in the toolbar above</li>
+        <li>Click <strong>Security Settings</strong></li>
+        <li>Turn the Slider to <strong>High</strong></li>
+        <li>If the page does not refresh automatically, <a href="/">click here</a> to refresh the page</li>
       </ol>
+
       <p><a href="/howto-disable-js">Not using the Tor Browser Bundle?</a></p>
```

Comment:
I'm trying to figure-out how to edit the latest version of the html files for just this PR, and once I'm able to access that, changing the text itself seems more reasonable and will be my objective. I did manage to at least get the latest files downloaded to my machine to run on Vagrant, but those include 0.4 fixes. 

"We recommend changing your Tor browser’s “Security Settings” slider to High to protect your anonymity. Learn how to disable it, or ignore this warning to continue."

It speaks to non-Tor users, and directs the user to the browser's menu text verbatim, within the onion button. Also, I want to put an image of the TBB into the message, and remove the "!" bang. Those signify a user error, and there is not a user error. Just an advisement. Combined, I believe these will be the most helpful. While also making the color purple and not red (but I assume that was already done, since the helpdoc image shows that).